lua/packer.lua

```diff
@@ -155,6 +218,7 @@ packer.make_commands = function()
   vim.cmd [[command! -nargs=* -complete=customlist,v:lua.require'packer'.plugin_complete PackerInstall lua require('packer').install(<f-args>)]]
   vim.cmd [[command! -nargs=* -complete=customlist,v:lua.require'packer'.plugin_complete PackerUpdate lua require('packer').update(<f-args>)]]
   vim.cmd [[command! -nargs=* -complete=customlist,v:lua.require'packer'.plugin_complete PackerSync lua require('packer').sync(<f-args>)]]
+  vim.cmd [[command! -nargs=* -complete=customlist,v:lua.require'packer.lockfile'.completion PackerLockfile lua require('packer').lockfile(<f-args>)]]
```

Comment:
To fix it, should this be 
```lua
vim.cmd [[command! -nargs=* -complete=customlist,v:lua.require'packer.lockfile'.plugin_complete PackerLockfile lua require('packer').lockfile(<f-args>)]]
```

Reply:
`-compele=` is for the command completion function it is not for the on packer complete autocommand.

The autocommand is fired from https://github.com/wbthomason/packer.nvim/blob/50aeb9060cf64c3c27e6d7b11d7af9e209ed6c3b/lua/packer.lua#L358-L360